src/util/audiosignal.h

```diff
@@ -70,7 +70,6 @@ class AudioSignal {
         DEBUG_ASSERT(kChannelCountZero <= m_channelCount); // unsigned value
         DEBUG_ASSERT(kSamplingRateZero <= m_samplingRate); // unsigned value
     }
-    virtual ~AudioSignal() {}
```

Comment:
C.35: A base class destructor should be either public and virtual, or protected and nonvirtual
http://isocpp.github.io/CppCoreGuidelines/CppCoreGuidelines#a-namerc-dtor-virtualac35-a-base-class-destructor-should-be-either-public-and-virtual-or-protected-and-nonvirtual